### DISCUSSION TITLE
Welcome to Beautiful Jekyll Discussions!

### DISCUSSION THREAD

**Discussion by daattali** [daattali/beautiful-jekyll#751] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you have any question, please [start a New Discussion](https://github.com/daattali/beautiful-jekyll/discussions/new)

**Comment by ghost** [comment]
I have a question. How do you add an Icon onto the website? This is because I want to add one on my template

**Reply by daattali** [reply]
First answer in the FAQ https://beautifuljekyll.com/faq/#favicon

**Comment by b-kennedy0** [comment]
Hello! I love your theme, thank you so much for the time you put into it. I'm in the process of trying to switch to using remote_theme + GitHub method so as to keep getting your updates!! You know your 'unreleased' section of the Changelog, when will those become released? My navbar dropdown text colour is now wrong 😢

**Reply by daattali** [reply]
There is no specific ETA. This is just one of about 20 unpaid projects I work on so I release whenever I find enough time to test it thoroughly and add all the new documentation. But you can still use the unreleased features - just don't add a version number. For example `remote_theme: daattali/beautiful-jekyll` will use the latest version on github

**Comment by daattali** [comment]
There's no parameter for that. If you really want to change it, you can
modify the CSS, but you'll have to modify a lot of rules because there are
many css properties associated with the navbar height

**Comment by morwarid1** [comment]
Hi, I want to use this template to create my portfolio webpage. For some reason, the webpage says error. How do I know I am searching the right URL for the page ?

**Reply by b-kennedy0** [reply]
Hi, what's the error _specifically_? 
Most are published at "username.GitHub.io" if user account.
Any extra detail can help

**Reply by morwarid1** [reply]
I followed the 3 steps mentioned in the Readme. I don't know what I am doing wrong.

**Reply by b-kennedy0** [reply]
Looks like you need to rename your repo. You currently have it as "mo.github.io" and you need to change it to your full username (morwarid1.GitHub.io)

**Reply by morwarid1** [reply]
Thanks!!!!! It worked

**Comment by CrhisBenitz** [comment]
Hi. I'm having an issue. I forked the repo and changed the name, but my Github Page site takes me to https://beautifuljekyll.com instead of the My website site shown on the tutorial. :( Do you know what's going on?

**Reply by b-kennedy0** [reply]
A couple of things.
- Your repository needs to be named "yourUsername.gitHub.io".
- It also looks as though you've deleted the index.html file and moved everything into a Docs folder, undo that. 
- You also need to state the url in the config.yml as "yourUsername.gitHub.io"

If that fails, refork from the original repo, and only make the changes suggested in the tutorial.

**Comment by jarjarwalker** [comment]
Hi, Im trying to change the avatar by adding an image to the _site/assets/img folder and pointing the avatar to the path of my image. Everytime I do that however and run bundle exec jekyll serve, my image is deleted from that folder
